config/default.yml

```diff
@@ -294,6 +294,12 @@ RSpec/DescribedClassModuleWrapping:
   VersionAdded: '1.37'
   Reference: https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/DescribedClassModuleWrapping
 
+RSpec/DescriptiveDescription:
```

Comment:
I suggest to follow the wip guideline for cop naming and make it reflect what it detects, `UndescriptiveDescription` or ‘UnclearDescription’.

Reply:
I updated this PR.